index.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>std-toast Styling Demos</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+    <!-- import the webpage's stylesheet -->
+    <link rel="stylesheet" href="/style.css">
+    <script type="module">
+      import 'std:elements/toast';
+    </script>
+    
+    <!-- import the webpage's javascript file -->
+    <script src="/script.js" defer></script>
+  </head>  
+  <body>
+    <style type="text/css" id="toast-styler">       
+    </style>
+    <h1>
```

Comment:
Nit: I think for headings `<h1>...</h1>` is easier to read than inserting two linebreaks and an indent.

Reply:
I 100% agree, this is the Glitch default though and typically I don't fight it. Changing all instances now